packages/simple-markdown/rollup.config.js

```diff
@@ -0,0 +1,10 @@
+export default {
```

Comment:
I don't think we need this config here. Our root rollup config should manage this.  If not, let's extend it to do the right thing for this package.